testing/pypy3/APKBUILD

```diff
@@ -0,0 +1,48 @@
+# Contributor: Mihnea Saracin <mihnea.saracin@rinftech.com>
+# Maintainer: Mihnea Saracin <mihnea.saracin@rinftech.com>
+pkgname=pypy3
+pkgver=5.7.1
+pkgrel=0
+pkgdesc="A fast, compliant alternative implementation of the Python 3 language"
+url="https://github.com/HorridBro/AlpinePyPy3"
+arch="all"
+license="MIT"
+depends=""
+makedepends="python linux-headers libffi-dev pkgconf zlib-dev bzip2-dev sqlite-dev ncurses-dev readline-dev libressl-dev expat-dev gdbm-dev paxmark tk tk-dev py3-cffi py2-cffi xz xz-dev"
```

Comment:
There’s no package named `python`, please replace it with `python2`.

Wrap this very long line at ~80 chars.